src/components/interactive-builder/modals/question/question-form/question-types/inputs/test-order/test-order-type-question.test.tsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import TestOrderTypeQuestion from './test-order-type-question.component';
+import { FormFieldProvider } from '../../../../form-field-context';
+import type { FormField } from '@openmrs/esm-form-engine-lib';
+
+// Mock the ConceptSearch component
+jest.mock('../../../common/concept-search/concept-search.component', () => {
+  return function MockConceptSearch({
+    defaultConcept,
+    onSelectConcept,
+  }: {
+    defaultConcept?: string;
+    onSelectConcept: (concept: { uuid: string; name: string }) => void;
+  }) {
+    return (
+      <input
+        data-testid="concept-search"
+        defaultValue={defaultConcept}
+        onChange={(e) => onSelectConcept({ uuid: e.target.value, name: e.target.value })}
+        placeholder="Search for a concept"
+      />
+    );
+  };
+});
+
+const mockSetFormField = jest.fn();
+const formField: FormField = {
+  id: 'test-order-1',
+  type: 'testOrder',
+  questionOptions: {
+    rendering: 'repeating',
+    concept: '',
+    selectableOrders: [],
+  },
+};
+
+jest.mock('../../../../form-field-context', () => ({
+  ...jest.requireActual('../../../../form-field-context'),
+  useFormField: () => ({ formField, setFormField: mockSetFormField }),
+}));
+
+describe('TestOrderTypeQuestion', () => {
+  it('renders the component correctly', () => {
+    renderTestOrderComponent();
+
+    expect(screen.getByText('Concept')).toBeInTheDocument();
```

Comment:
@RajPrakash681 your tests are failing because this text cannot be found. Since you have mocked the concept search input, you need to update the test so that it looks for the mocked input. You shouldn't need to mock the concept search, see this test for example - https://github.com/openmrs/openmrs-esm-form-builder/blob/09972c5c0048d4dbef7717d45526e42c08e5e270/src/components/interactive-builder/modals/question/question-form/question-types/inputs/obs/obs-type-question.test.tsx#L59-L69

Reply:
You can add `screen.logTestingPlaygroundURL();` before this line that will open up a testing playground where you can see what the test sees.